yoti-sdk-api/src/main/java/com/yoti/api/client/docs/DocScanService.java

```diff
@@ -317,6 +317,36 @@ public InstructionsResponse getIbvInstructions(String sdkId, KeyPair keyPair, St
         }
     }
 
+    public Media getIbvInstructionsPdf(String sdkId, KeyPair keyPair, String sessionId) throws DocScanException {
+        notNullOrEmpty(sdkId, "SDK ID");
+        notNull(keyPair, "Application key Pair");
+        notNullOrEmpty(sessionId, "sessionId");
+
+        String path = unsignedPathFactory.createFetchIbvInstructionsPdfPath(sdkId, sessionId);
+        LOG.info("Fetching instructions PDF at '{}'", path);
+
+        try {
```

Comment:
This appears to be getting repetitive, and the unit tests are becoming huge as a consequence.  Is there scope to introduce something reusable?